### ISSUE TITLE
hi buddy

### ISSUE THREAD

**Issue by mehmetkazim0** [FlisherOfatale/discord-auditlog#2]
Can I edit your module and make it Turkish? I'll add a link to your github on my own github.

**Comment by FlisherOfatale** [comment]
Hi, if you provided translated text, I could add some code to pick the "right" text from a localization file and open door to other languages as well.

**Comment by mehmetkazim0** [comment]
yes I can provide ma, it would even be great if you add some code
